cluster/ike-prow-template.yaml

```diff
@@ -71,6 +71,18 @@ objects:
             - name: sentry-dsn
               mountPath: /etc/sentry-dsn
               readOnly: true
+            readinessProbe:
+              httpGet:
+                path: /version
+                port: 8888
+              initialDelaySeconds: 5
+              periodSeconds: 5
+            livenessProbe:
+              httpGet:
+                path: /version
+                port: 8888
+              initialDelaySeconds: 5
+              periodSeconds: 5  
           volumes:
```

Comment:
Just wondering what stands behind the selection of these values? (initial delay and period)

Reply:
In other words why those numbers? :)